dxr/plugins/python/indexers.py

```diff
@@ -100,3 +233,95 @@ def _needle(self, needle_type, name, start, end):
                    Position(row=end[0],
                             col=end[1]))
         )
+
+    def get_bases_for_class(self, absolute_class_name):
+        """Return a list of all the classes that the given class
```

Comment:
Unfortunately, we lack `yield from`, so you'd need to make use of chain.from_iterable().